lib/restbase.js

```diff
@@ -41,6 +43,7 @@ function RESTBase (options, req) {
         this._parent = null;
         this._req = null;
         this._recursionDepth = 0;
+        this._isMain = true;
```

Comment:
Should set a _reqid to null here as well, in the same order as above to make sure that the object shape is the same.